server/googleActions.js

```diff
@@ -81,13 +96,11 @@ const GoogleActions = {
 
 	listGoogleBrews : async (req, res)=>{
 
-		oAuth2Client = GoogleActions.authCheck(req.account, res);
+		const oAuth2Client = GoogleActions.authCheck(req.account, res);
```

Comment:
`newGoogleBrew` has `auth` passed in externally from `homebrew.api` which uses `authCheck()` to generate the token. 

`listGoogleBrews` and `deleteGoogleBrew` still have the odd method of passing in the `req/res` objects instead and then running `authCheck` in here. Can we update them all to have the `authCheck` outside and oAuth token passed in? Or perhaps just pass in the `req.account` object? For consistency. Seems like errors can be thrown here, but actually adding errors to the to the `res` object should be happening in the api path.

I think making these consistent will help the next steps in the stubs update be cleaner.

Reply:
Yep I can do that and have the auth passed in and have it throw errors where applicable. FWIW I'm really thinking the auth check should be done as middleware and applied to the request for subsequent middlewares, but I'll do that in a separate PR.